proto/ha_scope.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package dash.ha_scope;
+
+import "types.proto";
+
+message HaScope {
+    // HA scope ID. It can be the HA set id (scope = dpu) or ENI id (scope = eni)
+    string ha_scope_id = 1;
```

Comment:
Is this field a duplicate of the information in the key? If so, can we remove it as this field isn't present in the HLD table schema

Reply:
addressed